package.json

```
@@ -1,5 +1,9 @@
 {
   "private": true,
+  "scripts": {
+    "clean": "rm -rf dist && rm -rf .cache",
+    "dev": "bin/parcel serve index.html"
```

Let's update the readme to call out at least this new command, instead of the manual call to `bin/parcel`.

---

Will do! It's still in progress 😄 Was going to finish the form first, but maybe I'll just push this up now.

---

For `package.json` scripts, they automatically look inside `node_modules/.bin`, so we can just do:

```suggestion
    "dev": "parcel serve index.html"
```